### DISCUSSION TITLE
安装 K8s 单机版 失败，网络插件cilium not running？

### DISCUSSION THREAD

**Discussion by opsmeta** [labring/sealos#4531] [Q&A]
SealosVersion:
  buildDate: "2023-07-17T10:32:54Z"
  compiler: gc
  gitCommit: 7ee53f1d
  gitVersion: 4.3.0
  goVersion: go1.20.5
  platform: linux/amd64

CentOS Linux release 7.9.2009 (Core)

日志r如下：
![image](https://github.com/labring/sealos/assets/65660520/58989bc0-1550-4456-9db7-13c6436182ef)

**Comment by bxy4543** [comment]
试试ubuntu2204
